processor/processorhelper/metadata.yaml

```diff
@@ -28,3 +28,13 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+
+    processor_duration:
+      enabled: true
+      stability:
+        level: alpha
+      description: Time taken by the processor to complete.
```

Comment:
nit: The description is not fully clear. Is it the time taken to process *one* item or one *batch* of items?